sekoia_automation/connector/event_aggregator.py

```diff
@@ -0,0 +1,187 @@
+import copy
+import datetime
+import threading
+import time
+from typing import Callable, Tuple
+
+import xxhash as xxhash
```

Comment:
```suggestion
import xxhash

Reply:
pycharm you are drunk. thank you @Darkheir

Reply:
fixed